components/Footer.js

```diff
@@ -1,7 +1,95 @@
 import React from "react";
+import {
+	FaDiscord,
+	FaInstagram,
+	FaSlack,
+	FaYoutube,
+	FaMediumM,
+	FaGithub,
+} from "react-icons/fa";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='bg-neutral-900 text-gray-50 py-6'>
+			<div className='grid sm:grid-rows-2 justify-center mx-auto'>
```

Comment:
Use React Bootstrap Grid with Row and Col components

Reply:
https://github.com/acm-ucr/acm-dev/blob/pondohoo/filter/components/Calendar.js 

 Check this example out!